static/xref/script.js

```diff
@@ -146,7 +148,15 @@ function howToCiteIDL(term, entry) {
   switch (type) {
     case 'exception':
       // Except for the following, exceptions take the form {{"SomeException"}}
-      if (!['EvalError', 'RangeError', 'ReferenceError', 'TypeError', 'URIError'].includes(term)) {
+      if (
+        ![
```

Comment:
Maybe we should make this a const above or outside the function entirely? Less ugly.